src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,51 @@
 package core.basesyntax.service;
 
-import core.basesyntax.dao.StorageDao;
-import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.dao.StorageDao;
+import core.basesyntax.db.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_PASSWORD_SIZE = 6;
+    private static final int MIN_AGE_FOR_REGISTER = 18;
+    private static final int MIN_AGE = 0;
```

Comment:
You minimal age for registration is 18, we don't need special check age below zero. Just make checking, for all ages bellow 18 didn't pass it.